build.sbt

```diff
@@ -121,8 +122,25 @@ def formattingPreferences = {
     .setPreference(SpacesAroundMultiImports, true)
 }
 
+val defaultMultiJvmOptions: List[String] = {
+  import scala.collection.JavaConverters._
+  // multinode.D= and multinode.X= makes it possible to pass arbitrary
+  // -D or -X arguments to the forked jvm, e.g. -Dmultinode.Xmx512m
```

Comment:
no, that example is correct, from `-Dmultinode.Xmx512m` it strips off `Dmultinode.` and what is passed to the forked jvm is `-Xmx512m`